src/components/20-molecules/datepicker/utils/date.js

```diff
@@ -161,9 +161,10 @@ const parseLocalisedDateIfValid = (
     )}-${addLeadingZeroes(day, 2)}T00:00:00`
   );
 
-  // IE 11 doesn't support Number.isNaN (bleeding-edge ES6)
-  // eslint-disable-next-line no-restricted-globals
-  const isValid = !isNaN(dateAsUnixEpochInteger);
+  // Check if its a valid number. Hint: empty string is number 0.
+  const isValid =
```

Comment:
Found this in an article on medium and I think this fit our needs: `const isNumberAlternative = value => typeof value === 'number' && !Number.isNan(value) && value !== Infinity && value !== -Infinity`

Reply:
https://medium.com/javascript-in-plain-english/how-to-check-for-a-number-in-javascript-8d9024708153